src/content/docs/browser/browser-monitoring/browser-pro-features/session-replay.mdx

```diff
@@ -316,7 +288,156 @@ To ensure your CSS files are accessible, add the `crossorigin="anonymous"` attri
+    2. Once the session is complete or consent is withdrawn, use `NREM.stopSessionReplay` to stop capturing data.
+* Record sessions based on specific criteria. For example:
+    * Start recording only for sessions on specific URLs such as a product checkout flow.
+    * Stop recording for sensitive pages such as `/billing` or `/health-records`.
+
+To ensure all user interactions are captured during manual recording, set sampling rates to 0% in your application settings.
+
+## Use Nerdgraph to configure replay settings [#nerdgraph-mutation]
+
+You can configure session replay settings using our [NerdGraph API](/docs/apis/nerdgraph/get-started/introduction-new-relic-nerdgraph/) instead of using the UI. This method is helpful if you need to make bulk changes, such as applying sampling rates to multiple browser entities.
+
+Here's an example mutation and variables to create a new browser application with default settings:
+
+<table>
+    <thead>
+        <tr>
+            <th>
+                Mutation
+            </th>
+            <th>
+                Variables
+            </th>
+        </tr>
+    </thead>
+    <tbody>
+        <tr>
+            <td>
+                ```graphql
+                mutation SessionReplayConfigUpdate($guid: EntityGuid!, $settings: AgentApplicationSettingsUpdateInput!) {
+                  agentApplicationSettingsUpdate(guid: $guid, settings: $settings) {
+                    browserSettings {
+                      sessionReplay {
+                        blockSelector
+                        errorSamplingRate
+                        maskTextSelector
+                        enabled
+                        collectFonts
+                        inlineImages
+                        inlineStylesheet
+                        samplingRate
+                        maskAllInputs
+                        maskInputOptions {
+                          color
+                          date
+                          datetimeLocal
+                          email
+                          month
+                          number
+                          range
+                          search
+                          select
+                          tel
+                          text
+                          textArea
+                          time
+                          url
+                          week
+                        }
+                      }
+                    }
+                    errors {
+                      field
+                      errorClass
+                      description
+                    }
+                  }
+                }
+                ```
+            </td>
+            <td>
+                ```json
+                {
+                  "sessionReplay": {
+                    "enabled": true,
+                    "collectFonts": true,
+                    "blockSelector": "String",
+                    "maskTextSelector": "String",
```

Comment:
Nobody would use "String" as their maskTextSelector. Let's use "*" (which is the default masking option)